pkg/v1/resource/certconfig/desired.go

```diff
@@ -199,6 +203,39 @@ func newCalicoCertConfig(clusterConfig cluster.Config, cert certs.Cert, projectN
 	}
 }
 
+func newClusterOperatorAPICertConfig(clusterConfig cluster.Config, cert certs.Cert, projectName string) *v1alpha1.CertConfig {
+	certName := string(cert)
+	return &v1alpha1.CertConfig{
+		TypeMeta: apimetav1.TypeMeta{
+			Kind:       certKind,
+			APIVersion: certAPIVersion,
+		},
+		ObjectMeta: apimetav1.ObjectMeta{
+			Name: key.CertConfigName(clusterConfig.ClusterID, cert),
+			Labels: map[string]string{
+				label.Cluster:         clusterConfig.ClusterID,
+				label.LegacyClusterID: clusterConfig.ClusterID,
+				label.LegacyComponent: certName,
```

Comment:
Do we need legacy labels here? Only cluster-operator with new certs library will use this cert, correct?

Reply:
Thanks, good point. Removed.